src/LSST/M1M3/TS/Settings/Setpoint.cpp

```diff
@@ -36,7 +36,7 @@ Setpoint::Setpoint(token) {
 }
 
 void Setpoint::load(YAML::Node doc) {
-    SPDLOG_INFO("Loading mixing valve settings");
+    SPDLOG_INFO("Loading Setpoint settigns.");
```

Comment:
```suggestion
    SPDLOG_INFO("Loading Setpoint settings.");
```

Am I missing something here? You do mean "settings," right?